config/index.js

```diff
@@ -17,6 +17,7 @@ exports.ADAPTERS_PATH = '/adapters';
 exports.ACTIONS_PATH = '/actions';
 exports.STATIC_PATH = 'static';
 exports.DEBUG_PATH = '/debug';
+exports.TEST_PATH = '/test';
```

Comment:
I originally had a test_controller assuming that the tests would be running out-of-process, but the tests run in-process so I removed the test_controller and missed removing this.